packages/blockchain-link/tsconfig.lib.json

```diff
@@ -1,20 +1,15 @@
 {
-    "compileOnSave": false,
+    "extends": "../../tsconfig.json",
     "compilerOptions": {
         "module": "commonjs",
         "target": "es2017",
-        "lib": ["dom", "es2019"],
+        "lib": ["webworker", "es2019"],
         "outDir": "./lib",
-        "strict": true,
-        "declaration": true,
-        "resolveJsonModule": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
-        "removeComments": true,
-        "types": ["jest", "node"],
-        "useUnknownInCatchVariables": false,
-        "allowSyntheticDefaultImports": true
+        "types": ["jest", "node", "web"],
+        "declarationMap": false,
+        "esModuleInterop": false,
+        "composite": false,
+        "incremental": false
     },
-    "include": ["./src/index.ts", "./src/utils/ws.ts", "./src/utils/socks-proxy-agent.ts"],
-    "exclude": ["node_modules"]
+    "include": ["./src"]
```

Comment:
or we can exclude `./src/ui` here since it's not needed in `lib` directory

Reply:
in the end, I decided to keep it excluded only in `files` in package.json because if I want to exclude it here I will have to create quite big `exclude` that will include `lib, libDev, nodeModules, src/ui, build` etc. because using exclude here will override inherited one